src/components/AddBlockModal.tsx

```diff
@@ -59,92 +60,156 @@ export const AddBlockModal: React.FC<AddBlockModalProps> = ({
       value: initialValues.value || "",
       controllingField: initialValues.controllingField || "",
     });
+    setErrors({});
   }, [initialValues]);
 
-  const isAlphabetic = (input: string) =>
-    input === "" || /^[a-zA-Z]+$/.test(input);
-
-  const validate = () => {
-    let valid = true;
-    let errors = { name: "", value: "", controllingField: "" };
-
-    if (!formData.name.trim()) {
-      errors.name = t("COMMON.REQUIRED_FIELD");
-      valid = false;
-    } else if (!isAlphabetic(formData.name)) {
-      errors.name = t("COMMON.INVALID_TEXT");
-      valid = false;
-    }
-
-    if (!formData.value.trim()) {
-      errors.value = t("COMMON.REQUIRED_FIELD");
-      valid = false;
-    } else if (!isAlphabetic(formData.value)) {
-      errors.value = t("COMMON.INVALID_TEXT");
-      valid = false;
-    }
+  useEffect(() => {
+    const fetchDistricts = async () => {
+      try {
+        const response = await getDistrictsForState({
+          limit: 10,
+          offset: 0,
+          controllingfieldfk: "MH",
```

Comment:
@rahulg1254 why are u hardcoding it here

Reply:
resolved